### ISSUE TITLE
Proper inheritance of Exceptions and stacktrace

### ISSUE THREAD

**Issue by cardil** [wavesoftware/javascript-eid-exceptions#7]
Now all exceptions have now the same prototype so it is possible to:

``` javascript
var e = new EidIllegalStateException('20160121:000629');
t instanceof EidNullPointerException; // returns true instead of false
```

Also stack trace is invalid:

``` javascript
var e = new EidIllegalStateException('20160121:000629');
e.stack // returns 
// Error
//     file.js:55
//     file.js:43
```

but should be:

``` javascript
var e = new EidIllegalStateException('20160121:000629');
e.stack // returns 
// EidIllegalStateException [20160121:000629]<d35vd8>
//     file.js:55
//     file.js:43
```

**Comment by cardil** [comment]
Fixed in PR #8

**Comment by cardil** [comment]
Refixed by #9
